lib/filter_lib/src/schema_generator.py

```diff
@@ -60,9 +60,29 @@ class _Sorts(GenericModel, Generic[TypeT]):
 
 
 class Pagination(BaseModel):
-    page_num: int = Field(1, gt=0)
+    page_num: Optional[int] = Field(None, gt=0, le=100)
```

Comment:
Why do you limit maximum `page_num`?

Reply:
fixed - this constraint shoud be applied to page_offset